src/main/java/org/opentripplanner/updater/vehicle_sharing/SharedVehiclesUpdater.java

```diff
@@ -16,19 +16,28 @@
 import org.opentripplanner.routing.graph.Vertex;
 import org.opentripplanner.updater.GraphUpdaterManager;
 import org.opentripplanner.updater.PollingGraphUpdater;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.Collection;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.stream.Stream;
 
 import static java.util.stream.Collectors.toList;
+import static org.apache.lucene.util.SloppyMath.haversin;
 
 public class SharedVehiclesUpdater extends PollingGraphUpdater {
-    SimpleStreetSplitter simpleStreetSplitter;
-    GraphUpdaterManager graphUpdaterManager;
-    VehiclePositionsGetter vehiclePositionsGetter;
-    Graph graph;
+
+    private static Logger LOG = LoggerFactory.getLogger(SharedVehiclesUpdater.class);
+
+    private static final double MAX_RADIUS = 500;
+    private static final double MAX_DISTANCE = 200;
```

Comment:
Jak to są dwie liczby, to jest trochę bez sensu. NAjpierw znajdujemy vertexy w promieniu 500 metrów, następnie zapuszczamy na tym filtry i reduce'y a potem robimy checka czy znaleziony node nie jest dalej niż 200 metrów.

Reply:
Te liczby nie są tym samym - pierwsza z nich to maksymalna dopuszczalna odległość do krawędzi, a druga to maksymalna dopuszczalna odległość do wierzchołka przy którym "zaparkujemy" pojazd. Nazwę je inaczej, ale moim zdaniem to powinny być dwie liczby. Wartości są do ustalenia jak dodamy faktyczne dane.